<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxPKUIClass</class>
 <widget class="QWidget" name="MessageBoxPKUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="coverIco">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/joy/message_default_ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="bodyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>360</width>
     <height>236</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="centerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>358</width>
      <height>228</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="messageLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>360</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102,102,102);
border-radius: 2px;
border: 2px solid #E6E6E6;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0,65,255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>bodyWidget</zorder>
  <zorder>coverIco</zorder>
 </widget>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
